WordPress/Classes/ViewRelated/Ratings/AppFeedbackPromptView.swift

```diff
@@ -44,6 +44,7 @@ class AppFeedbackPromptView: UIView {
         buttonStack.translatesAutoresizingMaskIntoConstraints = false
         buttonStack.axis = .horizontal
         buttonStack.spacing = LayoutConstants.buttonSpacing
+        buttonStack.isLayoutMarginsRelativeArrangement = true
```

Comment:
This was being called on `traitCollectionDidChange` every time so I moved it to the setup function.